torchserve_sanity.sh

```diff
@@ -35,7 +53,7 @@ stop_torchserve
 
 start_torchserve
 
-run_inference resnet-18 examples/image_classifier/kitten.jpg
```

Comment:
do you really want to remove this?
Wasn't this to verify snapshot resume?

Reply:
Done.